src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryTable.cs

```diff
@@ -32,7 +32,7 @@ public static async ValueTask<FormulaValue> LookUp(EvalVisitor runner, EvalVisit
                 else
                 {
                     var childContext = context.SymbolContext.WithScopeValues(row.Value);
-                    return await arg2.EvalAsync(runner, context.NewScope(childContext));
+                    return await arg2.EvalInRowScopeAsync(context.NewScope(childContext));
```

Comment:
>arg2.EvalInRowScopeAsync(context.NewScope(childContext)); [](http://example.com/codeflow?start=33&length=57)

Could this be
 arg2.EvalInRowScopeAsync(row);

and then line 34 is moved inside EvalInRowScopeAsync?

Reply:
As discussed offline, context may not always be created from RecordValue.